cli/README.md

```diff
@@ -0,0 +1,135 @@
+## Cook CLI
+
+### Installation
+
+To install the Cook CLI, clone this repo and from this folder run:
+
+```bash
+python3 setup.py install
+```
+
+This will install the `cs` command on your system.
+
+### Configuration
+
+The Cook CLI client is designed with "end user ergonomics" in mind. 
+Custom defaults may be provided for all commands. 
+Multiple clusters are supported via configuration.
+
+In order to use the Cook CLI, you’ll need a configuration file. 
+`cs` looks first for a `.cs.json` file in the current directory, and then for a `.cs.json` file in your home directory. 
+The path to this file may also be provided manually via the command line with the `--config` option.
+
+The file looks something like this:
+
+```json
+{
+  "defaults":{
+    "submit": {
+      "mem": 128,
+      "cpus": 1
+    }
+  },
+  "clusters": [
+    {
+      "name": "dev0",
+      "url": "http://127.0.0.1:12321/"
+    },
+    {
+      "name": "dev1",
+      "url": "http://127.0.0.1:12322/"
+    }
+  ]
+}
+```
+
+The `defaults` map contains default values that are passed each time a command is run. 
+For instance, the `submit` map above is used to set the defaults for `mem` and `cpus` when calling the `submit` command. 
+These can always be overridden directly from the command line. 
+For most commands, all available clusters are tried in order. 
+Each entry in the `clusters` array conforms to a cluster specification ("spec"). 
+A cluster spec requires a name and a url pointing to a Cook Scheduler.
+
+### Commands
+
+The fastest way to learn more about `cs` is with the `-h` (or `—help`) option.
```

Comment:
`–help` → `--help`
(the -- was converted to an en-dash)

Reply:
Done